pandas/tests/test_downstream.py

```diff
@@ -50,6 +50,7 @@ def test_dask(df):
     assert ddf.compute() is not None
 
 
+@pytest.mark.filterwarnings("ignore:Panel class is removed")
```

Comment:
Was this not causing CI failures before? Thought we failed on warnings like this

Reply:
this is showing up locally

Reply:
This should not be needed with latest xarray I suppose